thunder/core/prims.py

```diff
@@ -812,12 +820,21 @@ def _unpack_tuple_printer(
     if len(bsym.output) == 0:
         return f"# {call_str} (empty tuple)"
 
+    parts = [f"{codeutils.prettyprint(out, literals_as_underscores=True)}, " for out in out_printables]
```

Comment:
Should you put the added code into a function that is called from both places?

Reply:
It was on the edge of what I'd put in it's own functions, but yeah.